datajunction-clients/python/datajunction/nodes.py

```diff
@@ -18,6 +18,9 @@ class Namespace(ClientEntity):  # pylint: disable=protected-access
 
     namespace: str
 
+    #
+    # List
+    #
     def nodes(self):
```

Comment:
I know it's not part of this PR but I'm wondering if we should make this `list_nodes` to be consistent with the other client methods for getting a list of things.

Reply:
Agree. I'll walk through the Python client one more time this week.